src/storage/src/db_store.rs

```diff
@@ -487,13 +647,19 @@ impl DbStore {
             .execute_query(query)
             .map_err(|e| DB3Error::QueryKvError(format!("{}", e)))?;
         let mut values: Vec<_> = Vec::new();
+        let mut idx = 0;
         for op in ops.iter() {
             match op {
                 ProofOp::Push(Node::KV(k, v)) => {
                     let db3_doc = DB3Document::try_from(v.clone())?;
                     let doc = bson_util::bson_document_into_bytes(db3_doc.get_document()?);
                     let owner = db3_doc.get_owner()?.to_vec();
                     let tx_id = db3_doc.get_tx_id()?.as_ref().to_vec();
+
+                    if limit.is_some() && idx >= limit.unwrap() {
```

Comment:
the limit does not help too much

Reply:
yep just wanna support the feature. Will improve in the future. I guess we can do something inside the merk_db. e.g override execute_query(query) with limit rows